src/anemoi/graphs/create.py

```diff
@@ -55,13 +55,13 @@ def update_graph(self, graph: HeteroData) -> HeteroData:
             )
 
         for edges_cfg in self.config.get("edges", {}):
```

Comment:
```suggestion
        for edges_cfg in self.config.get("edges", {}):
 
            edge_builder_cfg = edges_cfg.get("edge_builder")
            if edge_builder_cfg is not None:
                edge_builder_cfg.source_mask_attr_name = edges_cfg.get("source_mask_attr_name")
                edge_builder_cfg.target_mask_attr_name = edges_cfg.get("target_mask_attr_name")
                edges_cfg.edge_builders = [edge_builder_cfg]



```

Reply:
This ensures backward compatibility of the config, which could be desirable since many users will not start using multiple builders. Possibly a depreciation warning can be added if the old config is not planned to remain supported.